### DISCUSSION TITLE
Assurance and coding information

### DISCUSSION THREAD

**Discussion by AmyAPP23** [marsupialproject/marsupial-emr#5] [General]
Firstly, thank you so much for sharing this wonderful software for health students. I am based in Australia and was actually working with a team to develop an app for our nursing students. After reviewing Marsupial emr, we are keen to investigate if we can integrate this into our nursing program. My IT division are seeking some assurance before we look at putting this on our networked computers. They have asked  me  to find out details of the developer/ marsupial project. They were unable to access source links on the release page for marsupial emr. There were some images and readme files rather than code. If this information is already on the site, can you please direct me to it. Looking forward to your response and thanks so much again.

**Comment by marsupialproject** [comment]
No problem! The software is not open source, so there is no publicly available code. As far as details about me, [I teach nursing at a public university in the United States](https://www.kent.edu/nursing/james-blackwell)

If there are concerns about security, I recommend running the software on computers that are not connected to an enterprise or domain network. Even as the developer, all the computers I use it on are not connected to my university's network. I have a separate network just for the simulation lab that is physically isolated from everything else. If they can't be physically isolated then the computers should be on their own virtual network (VLAN).

All the Marsupial software is digitally signed with an extended validation code signing certificate. You're free to run a virus or malware scan on the files.

**Comment by AmyAPP23** [comment]
Thank you for the information. Unfortunately, this is not going to meet our needs fully. All the best with your amazing work.
